packages/dashboard-core-plugins/src/panels/LogPanel.tsx

```diff
@@ -3,21 +3,36 @@
 import React, { PureComponent } from 'react';
 import { connect } from 'react-redux';
 import { LogView } from '@deephaven/console';
-import { GLPropTypes } from '@deephaven/dashboard';
-import { PropTypes as APIPropTypes } from '@deephaven/jsapi-shim';
+import { IdeSession } from '@deephaven/jsapi-shim';
 import Log from '@deephaven/log';
+import { Container, EventEmitter } from '@deephaven/golden-layout';
+import { RootState } from '@deephaven/redux';
 import './LogPanel.scss';
 import Panel from './Panel';
 import { getDashboardSessionWrapper } from '../redux';
 
 const log = Log.module('LogPanel');
 
-class LogPanel extends PureComponent {
+interface LogPanelProps {
+  glContainer: Container;
+  glEventHub: EventEmitter;
+  session: IdeSession;
```

Comment:
Technically this should be optional.

Doesn't matter except on the Enterprise side - we show a "Waiting for session connection..." message while the user has not yet connected.